src/messages/with_bytes.rs

```diff
@@ -139,18 +139,21 @@ mod tests {
     use crate::{
         id::FullId,
         location::SrcLocation,
-        rng::{self, MainRng},
+        rng::{self},
         unwrap,
     };
-    use rand::{self, Rng};
+    use rand::{self, distributions::Standard, Rng};
 
     #[test]
     fn serialise_and_partial_at_hop_message() {
         let mut rng = rng::new();
         let full_id = FullId::gen(&mut rng);
-        let msg = gen_message(&mut rng);
-
-        let signed_msg_org = unwrap!(SignedRoutingMessage::single_source(msg, &full_id,));
+        let msg = RoutingMessage {
+            src: SrcLocation::Node(*full_id.public_id().name()),
```

Comment:
Why the change from SrcLocation::Section to Node? Not sure it makes much difference but I wonder. (Particularly since all our hop messages have a Section as a source.

Reply:
I don't believe it makes any difference for this test, but I though that since the message is `single_source` it would be more realistic.